editor.planx.uk/src/ui/ExpandableList.tsx

```diff
@@ -41,20 +31,20 @@ export function ExpandableListItem(props: {
   headingId: string;
   groupId: string;
 }): FCReturn {
-  const classes = useItemClasses();
-
   const handleToggle = () => {
     props.onToggle && props.onToggle();
   };
 
   return (
-    <li
-      className={classnames(classes.root, props.expanded && classes.expanded)}
+    <ListItem
+      sx={{
+        borderBottom: (theme) => `1px solid ${theme.palette.secondary.main}`,
+        display: "block",
+      }}
```

Comment:
Just missing `padding: "0",` to prevent default padding on `<li>`

Reply:
Very good spot thank you 🏅